src/customConfigTests/src/test/resources/config.properties

```diff
@@ -1,6 +1,6 @@
-orderly.root=${orderly_test_root}
-db.template=${orderly_test_root}orderly.sqlite
-db.location=${orderly_test_root}testorderly.sqlite
+orderly.root=git/
```

Comment:
Have moved this over to use the same orderly directory that the server runs against, so that publishing works.